packages/world/mud.config.ts

```diff
@@ -103,6 +103,12 @@ export default mudConfig({
       },
       offchainOnly: true,
     },
+    WorldInitialized: {
+      keySchema: {},
+      valueSchema: {
+        isInitialized: "bool",
+      },
+    },
```

Comment:
this one got pulled out to the other PR right?

Reply:
i guess we can merge the other one first and then rebase this one

Reply:
Yes exactly, this PR should really not have those changes but it only occurred to me later to break it out